app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def calculate_team_total_rating(list_players: list) -> int | float:
```

Comment:
Use more descriptive type annotation. It should looks like `list_players: list[<class name here>]`. The same in all other similar cases

Reply:
fixed